tests/test_suites/ShortSystemTests/test_read_cache_low_level_percentage.sh

```diff
@@ -0,0 +1,25 @@
+timeout_set 10 minutes
+
+MOUNT_EXTRA_CONFIG="sfscachemode=NEVER`
+		`|readcachemaxsizepercentage=1`
+		`|maxreadaheadrequests=0`
+		`|cacheexpirationtime=10000" \
+    setup_local_empty_saunafs info
+
+cd "${info[mount0]}"
+
+# Calculate total system memory in MB
+total_memory_mb=$(free --mega | awk '/^Mem:/ {print $2}')
+
+# Calculate 2% of total memory
+two_percent_memory=$(echo "$total_memory_mb * 0.02" | bc)
```

Comment:
```suggestion
two_percent_memory=$(echo "${total_memory_mb} * 0.02" | bc | awk '{printf "%.0f", $0}')
```

Reply:
Consider rounding to integer using `awk` instead of `printf` since the latest doesn't handle floating point numbers very well by default and might fail.

Reply:
Done